qt-bugs@ issue : none
bugs.kde.org number : none
applied: no
author: from Trolltech

This patch fixes a rendering issue in bn_IN locale


--- src/kernel/qscriptengine_x11.cpp
+++ src/kernel/qscriptengine_x11.cpp

@@ -1514,7 +1514,7 @@
                         unsigned char pos = position[i];
                         for (int j = i+1; j > to+1; j--) {
                             uc[j] = uc[j-2];
-                            position[j] = uc[j-2];
+                            position[j] = position[j-2];
                         }
                         uc[to] = ch;
                         uc[to+1] = ch2;
